polls/models.py

```diff
@@ -0,0 +1,91 @@
+from django.db import models
+from django.utils import timezone
+
+from authentication.models import User
+
+
+class Poll(models.Model):
+    # The question of the poll.
+    question = models.CharField(
+        'question',
+        max_length=200,
+    )
+
+    # The user who created the poll.
+    user = models.ForeignKey(
+        User,
+        verbose_name='auteur',
+        on_delete=models.SET_NULL,
+        null=True,
+        related_name='polls',
+    )
+
+    # The date of creation.
+    creation_date_time = models.DateTimeField(
+        'date de création',
+        auto_now_add=True,
+    )
+
+    # The state of the poll.
+    STATES = (
+        ('REVIEWING', 'Validation en cours'),
+        ('REJECTED', 'Refusé'),
+        ('ACCEPTED', 'Accepté')
+    )
+    state = models.CharField(
+        'état',
+        max_length=9,
+        choices=STATES,
+        default='REVIEWING',
+    )
+
+    # The date of publication.
+    publication_date_time = models.DateTimeField(
```

Comment:
I think that having polls starting at midnight everyday and being available for one day is something a lot of people care about. Having an unique DateField to replace `publication_date_time` and `publication_end_date_time` will be easier to manage.